keyboards/handwired/wulkan/rules.mk

```
@@ -0,0 +1,25 @@
+MCU = STM32F303
+
+# Build Options
+#   comment out to disable the options.
+#
+BACKLIGHT_ENABLE = no
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no
+LAYOUTS = ortho_4x12
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
+NO_UART = yes
+NO_SUSPEND_POWER_DOWN = yes
+UNICODEMAP_ENABLE = yes
```

You may want to remove this, for the default, and only enable it for the keymaps that need it.

Aka, this is a big part of why Travis CI is failing hard. 

```suggestion
UNICODEMAP_ENABLE = no
```

---

Ah, missed that I could have a separate rules file for the keymaps, that is awesome!

---

Looks like Travis CI still is failing, but as what I can see it seems like it tries to apply all keymaps from handwired keyboards onto my keyboard and in most cases that just wont work

---

Nevermind my comment, seems to be just 3 keymaps that gives errors now and that is way better then before :)